# patch to remove hard-coded installation prefix
# Adam Huffman
# The Francis Crick Institute

diff -u bam-readcount-0.7.4/CMakeLists.txt bam-readcount-0.7.4.new/CMakeLists.txt
--- bam-readcount-0.7.4/CMakeLists.txt	2014-12-07 19:45:03.000000000 +0000
+++ bam-readcount-0.7.4.new/CMakeLists.txt	2016-04-06 22:06:12.336807391 +0100
@@ -2,7 +2,7 @@
 
 project(bam-readcount)
 
-set(CMAKE_INSTALL_PREFIX "/usr")
+#set(CMAKE_INSTALL_PREFIX "/usr")
 set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake;${CMAKE_SOURCE_DIR}/build-common/cmake")
 include(TestHelper)
 include(VersionHelper)
